raiden/utils/runnable.py

```diff
@@ -0,0 +1,71 @@
+from gevent import Greenlet
+from typing import Sequence
+
+
+class Runnable:
```

Comment:
I did it because I plan to move to true `Greenlet`s in the future. It requires refactoring and organizing of `cli.py`, `app.py` and `utils/cli.py`, then `App` may **always** be able to restart by being fully capable of reinstantiating all Runnables/Greenlets everywhere (including tests custom instantiations through custom `config`, smoketests, echo node, etc), and doing the required checks. `Runnable` is intended to be an almost drop-in replacement for `Greenlet` so we should be able to easily do the switch once we do [these refactors](https://github.com/raiden-network/raiden/pull/2208/files#r214356369)